### ISSUE TITLE
[Suggestion] add a default keybind for showing barriers or make them show by default

### ISSUE THREAD

**Issue by Thinkseal** [AmyMialeeMods/visiblebarriers#69]
Due to the fact that this mod does not have a config menu and that that is probably harder to do then this would probably be a good idea for there to be a default key to make barriers visible or make them show by default so people don't think them isn't working when they didn't look through the keybinds

**Comment by AmyMialee** [comment]
yeah it'll have a menu when i have the time to overhaul it, i'm just super busy
